src/generalChecks/constants.js

```diff
@@ -17,14 +17,26 @@ module.exports.generalMetrics = createObject([
   'CODE_OWNERS',
   'FOLDER_STRUCTURE',
   'BABEL_IMPORTS',
-  'CODE_QUALITY'
+  'CODE_QUALITY',
+  'I18N'
 ]);
 
 module.exports.DOCKERFILE_ATTRIBUTES = ['FROM', 'WORKDIR', 'COPY', 'RUN', 'ENV', 'WORKDIR'];
 
 module.exports.folderStructure = {
   react: ['app', 'config', 'constants', 'redux', 'scss', 'services', 'utils'],
-  vue: ['components', 'config', 'constants', 'scss', 'services', 'store', 'utils', 'views'],
+  vue: [
+    'assets',
+    'components',
+    'config',
+    'constants',
+    'mixins',
```

Comment:
I would remove this. It is not always neccesary

Reply:
Ok